### DISCUSSION TITLE
最高支持poc数量？

### DISCUSSION THREAD

**Discussion by AnQTroops** [projectdiscovery/nuclei#5384] [Q&A] [answered]
本地nuclei的poc数量8w+，使用-t指定poc目录，直接跑蹦。是否有解决方案？能否优化？至少需要支持数十万poc运行量~

**Comment by AnQTroops** [comment]
The poc number of the local nuclei is 8w+, use -t to specify the poc directory and run directly. Is there a solution? Can it be optimized? At least hundreds of thousands of poc runs need to be supported

**Comment by tarunKoyalwar** [comment, marked as answer]
@AnQTroops , 80k templates shouldn't be that big of a issue , this will directly affect start time of nuclei but will work , you need to mainly control two flags `-c` and `-bs`

```
-c -> no of templates in parallel
-bs -> no of targets in parallel per template
```

more info here https://docs.projectdiscovery.io/tools/nuclei/mass-scanning-cli

if you are facing issue or any type of crash for some reason you can always split the templates into multiple directories and run it one after another

```
/mytemplates/batch1 -> 10k
/mytemplates/batch2 -> 10k
....
```
